.github/workflows/tests.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       uses: codecov/codecov-action@v3
 
   qemu-tests:
-    runs-on: [ubuntu-22.04]
+    runs-on: [ubuntu-24.04]
```

Comment:
Be careful running tests on Ubuntu 24.04, if they're using our pytest_launcher code that uses `sys.exit` they will always pass (#2247).

Reply:
Is this the case just on the GitHub CI?

On the latest push, when I run `docker compose run -T ubuntu24.04 ./tests.sh -t qemu-user` locally, and intentionally make a test break (add assert False or something), then it does turn up red and the test shows as failing.

Reply:
I see - I can make fix proposed in #2247 in this PR.